source/element-parameters.md

```diff
@@ -3,15 +3,24 @@
 
 Lattice elements parameters are organized into **parameter groups**. 
 All groups are organized as abstract syntax trees.
-At the top level, there are the groups with names like `MagMultipole`, `ElecMultipole`, `RF`, `Alignment`, etc. 
+At the top level, there are the groups with names like 
+`MagMultipole`, `ElecMultipole`, `RF`, `Alignment`, etc. 
```

Comment:
I think it is better to be explicit in most cases here, even if it requires longer syntax.  For example, I would suggest `MagneticMultipole`, `ElectricMultipole`, etc.

Reply:
Done.